Cannot build Megit

**Source**: git clone of ffd52545e7de0f18773f71c7e09af83587273eef
**OS**: Devuan Daedalus
**Java/Javac**: 21
```
java --version
openjdk 21.0.5 2024-10-15 LTS
OpenJDK Runtime Environment Temurin-21.0.5+11 (build 21.0.5+11-LTS)
OpenJDK 64-Bit Server VM Temurin-21.0.5+11 (build 21.0.5+11-LTS, mixed mode, sharing)
```
```
javac --version
javac 21.0.5
```

**Maven**: 3.8.2
```
mvn --version
Apache Maven 3.8.2 (ea98e05a04480131370aa0c110b8c54cf726c06f)
Maven home: /home/ssarkar/Downloads/Software/apache-maven-3.8.2
Java version: 21.0.5, vendor: Eclipse Adoptium, runtime: /home/ssarkar-bkp/Software/jdk-21.0.5+11
Default locale: en_IN, platform encoding: UTF-8
OS name: "linux", version: "6.1.0-43-amd64", arch: "amd64", family: "unix"
```

Getting this error:
```
[WARNING] Error injecting: org.eclipse.tycho.p2resolver.DefaultTargetPlatformFactory
com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) No implementation for org.eclipse.tycho.core.resolver.MavenTargetLocationFactory was bound.
  at ClassRealm[extension>org.eclipse.tycho:tycho-maven-plugin:4.0.13, parent: jdk.internal.loader.ClassLoaders$AppClassLoader@639fee48] (via modules: org.eclipse.sisu.wire.WireModule -> org.eclipse.sisu.plexus.PlexusBindingModule)
  while locating org.eclipse.tycho.p2resolver.TargetDefinitionResolverService
  while locating org.eclipse.tycho.p2resolver.DefaultTargetPlatformFactory
```

Full log has been attached.

Tried deleting tycho from local Maven repo, then retry, same result.

[log.txt](https://github.com/user-attachments/files/25568066/log.txt)

With Maven 3.9.12, I get the following error:

```
mvn install
[INFO] Scanning for projects...
[INFO] Tycho Version:  4.0.13 (dbd40458ea67f56274c25d9d1fd7f2e5af8971b0)
[INFO] Tycho Mode:     project
[INFO] Tycho Builder:  maven
[INFO] Build Threads:  1
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] com.eclipsesource.megit.parent                                     [pom]
[INFO] com.eclipsesource.megit.plugin                          [eclipse-plugin]
[INFO] com.eclipsesource.megit.product                     [eclipse-repository]
[INFO] 
[INFO] -------< com.eclipsesource.megit:com.eclipsesource.megit.parent >-------
[INFO] Building com.eclipsesource.megit.parent 0.10.0-SNAPSHOT            [1/3]
[INFO]   from pom.xml
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- tycho-source:4.0.13:plugin-source (plugin-source) @ com.eclipsesource.megit.parent ---
[INFO] 
[INFO] --- install:3.1.2:install (default-install) @ com.eclipsesource.megit.parent ---
[INFO] Installing /home/ssarkar-bkp/Software/Java/megit/com.eclipsesource.megit.parent/pom.xml to /mnt/home/ssarkar/.m2/repository/com/eclipsesource/megit/com.eclipsesource.megit.parent/0.10.0-SNAPSHOT/com.eclipsesource.megit.parent-0.10.0-SNAPSHOT.pom
[INFO] 
[INFO] -------< com.eclipsesource.megit:com.eclipsesource.megit.plugin >-------
[INFO] Building com.eclipsesource.megit.plugin 0.10.0-SNAPSHOT            [2/3]
[INFO]   from /home/ssarkar-bkp/Software/Java/megit/com.eclipsesource.megit.plugin/pom.xml
[INFO] ---------------------------[ eclipse-plugin ]---------------------------
[INFO] Resolving target definition file:/home/ssarkar-bkp/Software/Java/megit/com.eclipsesource.megit.product/../com.eclipsesource.megit.parent/megit.target for environments=[win32/win32/x86_64, linux/gtk/x86_64, linux/gtk/aarch64, macosx/cocoa/x86_64, macosx/cocoa/aarch64], include source mode=honor, referenced repository mode =ignore, execution environment=org.eclipse.tycho.core.ee.impl.CustomEEResolutionHints@34fa8304 with Tycho Provisioning Agent (extension>org.eclipse.tycho:tycho-maven-plugin:4.0.13)
[INFO] ### Using TychoRepositoryTransport for remote P2 access ###
[INFO]     Cache location:         /mnt/home/ssarkar/.m2/repository/.cache/tycho
[INFO]     Transport mode:         online
[INFO]     Http Transport type:    Java11Client
[INFO]     Update mode:            cache first
[INFO]     Minimum cache duration: 60 minutes
[INFO]       (you can configure this with -Dtycho.p2.transport.min-cache-minutes=<desired minimum cache duration>)
[INFO] Downloading from p2: https://download.eclipse.org/releases/2025-12/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/releases/2025-12/p2.index (122 bytes at 0 bytes/s)
[INFO] Downloading from p2: https://download.eclipse.org/technology/epp/packages/2025-12/p2.index
[INFO] Downloading from p2: https://download.eclipse.org/technology/epp/packages/2025-12/202512041000/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/technology/epp/packages/2025-12/202512041000/p2.index (365 bytes at 356 KB/s)
[INFO] Downloading from p2: https://download.eclipse.org/releases/2025-12/202512101000/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/releases/2025-12/202512101000/p2.index (140 bytes at 136 KB/s)
[INFO] Downloading from p2: https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2025-12/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2025-12/p2.index (126 bytes at 123 KB/s)
[INFO] Downloading from p2: https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/release/4.38.0/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/release/4.38.0/p2.index (172 bytes at 167 KB/s)
[INFO] Downloading from p2: https://download.eclipse.org/justj/jres/21/updates/release/latest/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/justj/jres/21/updates/release/latest/p2.index (126 bytes at 123 KB/s)
[INFO] Downloading from p2: https://download.eclipse.org/justj/jres/21/updates/release/21.0.10/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/justj/jres/21/updates/release/21.0.10/p2.index (172 bytes at 167 KB/s)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for com.eclipsesource.megit.parent 0.10.0-SNAPSHOT:
[INFO] 
[INFO] com.eclipsesource.megit.parent ..................... SUCCESS [  0.181 s]
[INFO] com.eclipsesource.megit.plugin ..................... FAILURE [  4.959 s]
[INFO] com.eclipsesource.megit.product .................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  14.923 s
[INFO] Finished at: 2026-02-26T12:42:41+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to resolve target definition file:/home/ssarkar-bkp/Software/Java/megit/com.eclipsesource.megit.product/../com.eclipsesource.megit.parent/megit.target: Could not find "org.eclipse.justj.openjdk.hotspot.jre.full.feature.group/21.0.9.v20251105-0741" in the repositories of the current location
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
```

No target is generated.

--- a/com.eclipsesource.megit.parent/megit.target
+++ b/com.eclipsesource.megit.parent/megit.target
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/eclipse-cbi/targetplatform-dsl -->
-<target name="MeGit Target" sequenceNumber="1766789774">
+<target name="MeGit Target" sequenceNumber="1772092922">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
       <unit id="org.eclipse.jgit.feature.group" version="7.5.0.202512021534-r"/>
@@ -149,7 +149,7 @@
       <repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2025-12"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.justj.openjdk.hotspot.jre.full.feature.group" version="21.0.9.v20251105-0741"/>
+      <unit id="org.eclipse.justj.openjdk.hotspot.jre.full.feature.group" version="21.0.10.v20260205-0638"/>
       <repository location="https://download.eclipse.org/justj/jres/21/updates/release/latest"/>
     </location>
     <location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" missingManifest="error" type="Maven" label="extras-megit">

